.github/workflows/build_deploy.yml

```diff
@@ -37,7 +37,9 @@ jobs:
     # Runs a set of commands using the runners shell
     - name: Run a multi-line script
       run: |
-        git submodule update --init --recursive
+        curl -O -sSL https://github.com/ReproNim/reproschema-ui/archive/8519d6141f68f277f1e36c1b1be62810bb1bdc93.zip
+        unzip 8519d6141f68f277f1e36c1b1be62810bb1bdc93.zip
```

Comment:
do you want to use this version of schema ui? or the current master checksum: https://github.com/ReproNim/reproschema-ui/tree/0ffbfe965e2931b93a3efc620b14551d29e5707a

Reply:
yes, the current master. I'll update it. how do you get that URL with the checksum?